lib/tutorials/en_US/plugins.md

```diff
@@ -1,205 +1,166 @@
 
-api.route({
-    method: 'GET',
-    path: '/',
-    handler: function (request, reply) {
-        reply('api index');
-    }
-});
-```
+### The register method
 
-Multiple labels can be selected at the same time by passing an array of strings, this works as a logical OR statement. To accomplish a logical AND, you can chain calls to `server.select()` like so:
+As we've seen above, the `register` method accepts two parameters, `server` and `options`.
 
-```javascript
-// all servers with a label of backend OR api
-const myServers = server.select(['backend', 'api']);
+The `options` parameter is simply whatever options the user passes to your plugin. No changes are made and the object is passed directly to your `register` method.
 
-// servers with a label of api AND admin
-const adminServers = server.select('api').select('admin');
-```
+`register` should be an async function that returns once your plugin has completed whatever steps are necessary for it to be registered. Alternatively your register plugin should throw an error if an error occurred while registering your plugin.
 
-The return value of `server.select()` is a server object that contains only the selected connections.
+The `server` object is a reference to the `server` your plugin is being loaded in.
 
 ## Loading a plugin
 
 Plugins can be loaded one at a time, or as a group in an array, by the `server.register()` method, for example:
 
 ```javascript
-// load one plugin
-server.register(require('myplugin'), (err) => {
-    if (err) {
-        console.error('Failed to load plugin:', err);
-    }
-});
+const start = async function () {
 
-// load multiple plugins
-server.register([require('myplugin'), require('yourplugin')], (err) => {
-    if (err) {
-        console.error('Failed to load a plugin:', err);
-    }
-});
+    // load one plugin
+
+    await server.register(require('myplugin'));
+
+    // load multiple plugins
+
+    await server.register([require('myplugin'), require('yourplugin')]);
+};
 ```
 
-To pass options to your plugin, we instead create an object with `register` and `options` keys, such as:
+To pass options to your plugin, we instead pass an object with `register` and `options` keys, such as:
 
 ```javascript
-server.register({
-    register: require('myplugin'),
-    options: {
-        message: 'hello'
-    }
-}, (err) => {
+const start = async function () {
 
-    if (err) {
-        throw err;
-    }
-});
+    await server.register({
+        register: require('myplugin'),
+        options: {
+            message: 'hello'
+        }
+    });
+};
 ```
 
 These objects can also be passed in an array:
 
 ```javascript
-server.register([{
-    register: require('plugin1'),
-    options: {}
-}, {
-    register: require('plugin2'),
-    options: {}
-}], (err) => {
-
-    if (err) {
-        throw err;
-    }
-});
+const start = async function () {
+
+    await server.register([{
+        register: require('plugin1'),
```

Comment:
Ditto here on the property name changing from `register` to `plugin`—I wont mark any others, but there are likely more.